cmd/app/options/options.go

```diff
@@ -221,4 +225,9 @@ func (o *Options) addControllerFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.Controller.LeaderElectionNamespace,
 		"leader-election-namespace", "istio-system",
 		"Namespace to use for controller leader election.")
+
+	fs.StringVar(&o.Controller.NamespaceSelector,
+		"namespace-selector", "",
```

Comment:
Small nit: can we change `namespace-selector` to `configmap-namespace-selector` to make it more explicit what we are doing with these namespaces at glance?

Reply:
@JoshVanL The parameter is renamed. Thanks for the suggestion.